### PR TITLE
Re-enable package-lock.json

### PR DESCRIPTION
**Pull request by dpgraham**
## Proposed changes

It turns out that disabling package-lock.json _also_ disables npm-shrinkwrap.json, which means that users who have been installing Appium using npm >= 5 have been getting a non-shrinkwrapped version of Appium.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by jlipps** [comment]
should we do this or just run npm shrinkwrap as part of the publish step?

**Comment by dpgraham** [comment]
The issue is that when `package-lock=false`, if you install Appium it will ignore `npm-shrinkwrap.json`

File: .npmrc [hunk 1]
```diff
@@ -1 +1,2 @@
-package-lock=false
\ No newline at end of file
+# Do not disable package-lock. Disabling package-lock also disables npm-shrinkwrap
+# package-lock=false
```
Comment (imurchie) [inline .npmrc]: I think this should be `package-lock=true` rather than commented out.

**Review by imurchie — APPROVED** [review]
Nice of npm to have these undocumented effects.

**Pull request merged**